### ISSUE TITLE
If TestsFolder contains a space an error is thrown

### ISSUE THREAD

**Issue by tomaustin700** [pester/AzureDevOpsExtension#53]
### Where are you running it?
- Azure DevOps Service (VSTS)

### Version of Extension/Task
Current version

### Expected behaviour and actual behaviour
If the TestFolder variable uses a path that contains a space in it then the Pester test runner will fail. For example my self-hosted agents are installed the the following directory C:\Build Agent\ so when I use $(System.DefaultWorkingDirectory)\* it is evaluated to C:\Build Agent\_work\27\s\\*, this then throws the following error
```
Using executable 'powershell.exe'
powershell.exe C:\Build Agent\_work\_tasks\Pester_cca5462b-887d-4617-bf3f-dcf0d3c622e9\10.2.4\Pester.ps1 -TestFolder C:\Build Agent\_work\26\s\* -resultsFile C:\Build Agent\_work\26\s\Test-Pester.XML -run32Bit False
##[error]C:\Build : The term 'C:\Build' is not recognized as the name of a cmdlet, function, script file, or operable program. 

##[error]C:\Build : The term 'C:\Build' is not recognized as the name of a cmdlet, function, script file, or operable program. 

##[error]Check the spelling of the name, or if a path was included, verify that the path is correct and try again.
```

### Steps to reproduce the problem
Use a TestsFolder path that contains a space in the path

**Comment by ChrisLGardner** [comment]
I can see where in our code this is happening so I'll have to investigate ways to solve this. Hopefully I'll have a fix out in the next few days for this.

**Comment by ChrisLGardner** [comment]
Should now be resolved in 10.3.x and 9.4.x
